### PR TITLE
fix 246

### LINKED ISSUE #246: Imports require whole name for AW functions
For API calls we support both `FindNextFileA` and `FindNextFile`, but for imports we only support `FindNextFileA`. I think this is confusing. Should we use the `generate_api_features` helper for imports as well?

### PR DESCRIPTION
**Pull request by williballenthin**
add helper to generate import features with and without the A/W suffixes. 

closes #246 (import features without A/W)

### CODE REVIEW

File: capa/features/extractors/helpers.py [hunk 1]
```diff
@@ -65,6 +66,34 @@ def generate_api_features(apiname, va):
...
+    these include:
+      - kernel32.CreateFileA
+      - kernel32.CreateFile
+      - CreateFileA
+      - CreateFile
+    """
+    # (kernel32.CreateFileA, 0x401000)
+    yield Import(dll + "." + symbol), va
+    # (CreateFileA, 0x401000)
+    if not is_ordinal(symbol):
```
Comment (Ana06) [inline capa/features/extractors/helpers.py:86]: Isn't something like this also needed in the api function? I think `generate_api_features()` is currently it is returning things like `ord3` for visisect.

**Review by Ana06 — APPROVED** [review]
LGTM :+1: Although, I would aim to unify the implementations of `generate_import_features` and `generate_api_features`.

**Review by mr-tz — APPROVED** [review]
see @Ana06's comments, otherwise looks great.

File: capa/features/extractors/helpers.py [hunk 2]
```diff
@@ -65,6 +66,34 @@ def generate_api_features(apiname, va):
             yield API(impname[:-1]), va
 
 
+def is_ordinal(symbol):
+    return symbol[0] == "#"
+
+
+def generate_import_features(dll, symbol, va):
```
[the remark this answers is not in the document]
Comment (williballenthin) [reply capa/features/extractors/helpers.py:73]: ack, good point!

**Pull request merged**